src/framegrab/grabber.py

```diff
@@ -10,30 +10,43 @@
 import cv2
 import numpy as np
 
+logger = logging.getLogger(__name__)
+
 # Optional imports
 try:
     from pypylon import pylon
-except ImportError:
+except ImportError as e:
+    logger.warning(
```

Comment:
The message is clear, but it will still be annoying for far more people than it is helpful.  

Instead I suggest using [UnavailableModule](https://github.com/groundlight/python-sdk/blob/1fc8ec847470ed6a59d3c1a8c214edd60691dff8/src/groundlight/optional_imports.py#L13) - just copy it in here.